external/storm-jdbc/src/test/java/org/apache/storm/jdbc/common/JdbcClientTest.java

```diff
@@ -80,6 +85,24 @@ public void testInsertAndSelect() {
         Assert.assertEquals(rows, selectedRows);
     }
 
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testInsertConnectionError() {
+
+        ConnectionProvider connectionProvider = new MockConnectionProvider(null);
+        this.client = new JdbcClient(connectionProvider, 60);
+
+        List<Column> row = createRow(1, "frank");
+        List<List<Column>> rows  = new ArrayList<List<Column>>();
+        rows.add(row);
+        String query  = "insert into user_details values(?,?,?)";
+
+        thrown.expect(MultipleFailureException.class);
```

Comment:
I am not super familiar with MultipleFailureException.  I though that `thrown.expect(MultipleFailureException.class)` will verify that an instance of MultipleFailureException is thrown by `client.executeInsertQuery...` but because it is not a part of our code I must be wrong.  Is it being treated like a wild card then?